src/Language/Fixpoint/Smt/Interface.hs

```diff
@@ -254,18 +254,20 @@ negativeP
   = do v <- A.char '(' *> A.takeWhile1 (/=')') <* A.char ')'
        return $ "(" <> v <> ")"
 
-smtWriteRaw      :: Context -> Raw -> IO ()
+-- | Writes a line of input for the SMT solver and to the log if there is one.
+smtWriteRaw :: Context -> Raw -> IO ()
 smtWriteRaw me !s = {- SCC "smtWriteRaw" -} do
   -- whenLoud $ do LTIO.appendFile debugFile (s <> "\n")
   --               LTIO.putStrLn ("CMD-RAW:" <> s <> ":CMD-RAW:DONE")
-  hPutStrLnNow (ctxCout me) s
+  hPutStrLnNow (ctxIn me) s
   maybe (return ()) (`LTIO.hPutStrLn` s) (ctxLog me)
 
-smtReadRaw       :: Context -> IO T.Text
-smtReadRaw me    = {- SCC "smtReadRaw" -} TIO.hGetLine (ctxCin me)
```

Comment:
Can I remove `{- SCC "smtReadRaw" -}`? It is there as a pragma on the next line.

Reply:
I think so.